options.go

```diff
@@ -37,3 +38,10 @@ func Filter(filter map[string]string) Option {
 		args.Filter = filter
 	}
 }
+
+// Decoder is available if a custom decoder function must be used when reading the SQS queue
+func WithDecoder(decoder func(*string) ([]byte, error)) Option {
```

Comment:
This name seems not consistent with other option names, due to this I more like previous solution

Reply:
Ah, that was my bad. I suggested this change because I'm used to this format from the options here: https://godoc.org/github.com/grpc-ecosystem/grpc-gateway/runtime#ServeMuxOption, and didn't realize it didn't conform with the ones already here.